### DISCUSSION TITLE
How to move a file to subfolder?

### DISCUSSION THREAD

**Discussion by Nummuliitti** [community/community#109550] [New to GitHub]
Hello.

I have a local repository : "D:\test\src" that has two files "app.jsx" and "main.jsx"

These two files must be in this directory locally, in order for the code to work. A program looks them up in this folder.

I have created a subfolder "/osa1" in my github.com repository.

How can i move these two files into this subfolder, but still keep the local repository path intact?

Please see attached picture that explains it.

Thank you!

![subfolder](https://github.com/community/community/assets/159144494/9b3711b3-195b-47bb-af42-cf7f98905696)

**Comment by NipulM** [comment]
Hello 👋🏼

In order to this, you first need to clone the GitHub repository locally.

Clone the GitHub repository locally:
`git clone <URL_OF_THE_REPO>`

Go to the main directory:
`cd D:\test\src`

Create the 'osa1' subfolder and move the files to it:
```
mkdir D:\test\src\osa1
move D:\test\src\app.jsx D:\test\src\osa1
move D:\test\src\main.jsx D:\test\src\osa1
```

Once you've completed the above steps, go back to the root directory:
`cd D:\test\src`

Add the changes, commit, and push to the GitHub repository:
```
git add .
git commit -m "<COMMIT_MSG>"
git push origin main
```
This should do the trick 🙌🏼
Please make sure to replace <COMMIT_MSG> and <URL_OF_THE_REPO> with your own commit message and repository URL, respectively.

**Comment by Nummuliitti** [comment]
Hello

The two files "app.jsx" and "main.jsx" MUST be locally in the folder D:/test/src. I cannot move them into a subfolder D:/test/src/osa1

The reason is because a program looks these two files up in this directory. They cannot be moved.

How to proceed?
